src/components/views/rooms/LinkPreviewWidget.js

```diff
@@ -18,7 +18,9 @@ limitations under the License.
 import React, {createRef} from 'react';
 import PropTypes from 'prop-types';
 import createReactClass from 'create-react-class';
-import { linkifyElement } from '../../../HtmlUtils';
+import { AllHtmlEntities } from 'html-entities';
+
```

Comment:
why the break in the imports? This is one of those tiny things that is going to make merge conflicts a nightmare.